app/scripts/components/analysis/results/use-analysis-params.ts

```diff
@@ -25,8 +25,8 @@ type AnyAnalysisParamsKey = keyof AnalysisParams;
 type AnyAnalysisParamsType = Date | DatasetLayer[] | FeatureCollection<Polygon>;
 
 const initialState: AnalysisParamsNull = {
-  start: undefined,
-  end: undefined,
+  start: new Date(2018, 0, 1),
+  end: new Date(2022, 11, 31),
```

Comment:
```suggestion
  end: new Date(2022, 12, 31),
```

Reply:
@j08lue No, in JS the date constructor takes a 0-indexed month.

Reply:
oops, sorry

Reply:
wait, what?!

Reply:
See https://stackoverflow.com/questions/2552483/why-does-the-month-argument-range-from-0-to-11-in-javascripts-date-constructor